src/main/java/uk/gov/hmcts/reform/sscs/ccd/service/CcdService.java

```diff
@@ -188,14 +187,24 @@ private SscsCaseDetails getCaseByAppealNumber(String appealNumber, IdamTokens id
         List<SscsCaseDetails> caseDetailsList = searchCcdCaseService.findCaseBySearchCriteria(ImmutableMap.of(
             "case.subscriptions.appellantSubscription.tya", appealNumber), idamTokens);
 
+        caseDetailsList = caseDetailsList.stream()
```

Comment:
Just a suggestion but you could probably move this block to below the 2 other if statements, so you only need to have it in the code once rather than 3 times

Reply:
Oh yes, true. It's updated